HelloWorld/HelloWorld.cs

```diff
@@ -6,7 +6,23 @@ class Program
     {
         static void Main(string[] args)
         {
-           Console.WriteLine("Hello Drew!");
+
+        // project section of first lesson
+        Console.WriteLine("Hello Drew!");
+
+        // homework section of first lesson
+        string name = "";
+		int age = 0;
+		int year = 0;
+
+        Console.WriteLine("Please enter your name: ");
+		name = Console.ReadLine();
+		Console.WriteLine("Please enter your age: ");
+		age = Convert.ToInt32(Console.ReadLine());
+		Console.WriteLine("Please enter the year: ");
```

Comment:
Put on your devil's advocate hat and think about how this question might cause problems. If you want to print out the year someone was born. What happens when the user (the person interacting with your application) is has a birthday after the day they are using your application? What about before?